frontend/packages/console-dynamic-plugin-sdk/src/api/api.ts

```diff
@@ -9,3 +14,4 @@ const MockImpl = () => {
 export const useK8sWatchResource: UseK8sWatchResource = MockImpl;
 export const useK8sWatchResources: UseK8sWatchResources = MockImpl;
 export const useResolvedExtensions: UseResolvedExtensions = MockImpl;
+export const HorizontalNav: React.FC<HorizontalNavProps> = MockImpl;
```

Comment:
Please add JS doc for this component.

Reply:
Is it okay if I raise a seperate PR for JSDoc? It will require some configuration code as well.

Reply:
I'd suggest making any changes to the config / docs script in a separate PR, but it would be good to add the JSDoc comment itself as part of this change.